.github/workflows/ci.yml

```diff
@@ -25,7 +25,7 @@ jobs:
           submodules: recursive
       - run: rustup update ${{ matrix.toolchain }} && rustup default ${{ matrix.toolchain }}
       - run: rustup target add wasm32-unknown-unknown
-      - run: cargo install --locked soroban-cli --features opt
+      - run: cargo install --locked --version 20.3.4 soroban-cli --features opt
```

Comment:
@CristhianRodriguezMolina Is it really necessary to specify the CLI version?

Reply:
I think we don't really need it :thinking: